content/vendor/guides/quickstart/_index.md

```diff
@@ -1,7 +1,7 @@
 ---
 date: "2020-02-20T00:00:00Z"
 lastmod: "2020-02-20T00:00:00Z"
-title: "Quickstart"
-redirect: "/vendor/quickstart/create-release"
+title: "Guides"
+redirect: "/vendor/guides/quickstart/create-release"
```

Comment:
@dexhorthy will the current URL be redirected so we don't get 404s?

Reply:
good catch, yeah let me update that. thanks.

Reply:
done